app/services/FilterService.scala

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import config.AppConfig
+import models.getPenaltyDetails.GetPenaltyDetails
+import models.getPenaltyDetails.appealInfo.AppealStatusEnum
+import models.getPenaltyDetails.latePayment.{LPPDetails, LPPPenaltyCategoryEnum, LPPPenaltyStatusEnum, LatePaymentPenalty}
+import models.getPenaltyDetails.lateSubmission.{LSPDetails, LateSubmissionPenalty}
+import play.api.libs.json.{JsString, JsValue, Json}
+import utils.Logger.logger
+
+import javax.inject.Inject
+import scala.concurrent.ExecutionContext
+import scala.util.Try
+
+class FilterService @Inject()()(implicit ec: ExecutionContext, appConfig: AppConfig) {
+
+  def tryJsonParseOrJsSting(body: String): JsValue = {
+    Try(Json.parse(body)).getOrElse(JsString(body))
+  }
+
+  def filterEstimatedLPP1DuringPeriodOfFamiliarisation(penaltiesDetails: GetPenaltyDetails, callingClass: String, function: String, vrn: String): GetPenaltyDetails = {
+    if (penaltiesDetails.latePaymentPenalty.nonEmpty) {
+      val filtered: Option[Seq[LPPDetails]] = filterEstimatedLPP1(penaltiesDetails)
+      val numberOfFiltered = numberOfFilteredLPPs(filtered, penaltiesDetails)
```

Comment:
Setting these to filteredLPPs would help with readability and maybe saying that they are filtered due to being LPP1's in POF